docs/ios/apple-account-management.md

```diff
@@ -6,53 +6,63 @@ ms.date: 11/08/2024
+If you have enrolled in Apple's **Enterprise Program**, you will need to:
 
-1. To add an individual account, in the **Individual Account...** dialog, enter your App Store Connect API key data and click the **Add** button:
+1. Sign-in to your [Apple Developer Account](https://developer.apple.com/account)
+2. Select **Users and Access** located under the **Services** section
+3. Select the **Integrations** tab
+4. Click the **+** button
 
-    :::image type="content" source="media/apple-account-management/vs/individual-account.png" alt-text="Add an Individual Apple Developer Account to Visual Studio.":::
+> [!NOTE]
+> Only **Admin** keys will be able to register new Bundle IDs and generate new provisioning profiles. **Developer** keys will not be able to perform these tasks.
+>
+> For more information about how to generate an Enterprise API Key, visit Apple's [Enterprise Program API documentation](https://developer.apple.com/documentation/enterpriseprogramapi).
 
-    The **Name**, **Issuer ID**, and **Key ID** data can be found in [App Store Connect](https://appstoreconnect.apple.com/) by selecting **Users and Access** and then the **Keys** tab. The **Private key** can also be downloaded from this location:
+## Add an Apple Developer Account
 
-    :::image type="content" source="media/apple-account-management/app-store-connect-details.png" lightbox="media/apple-account-management/app-store-connect-details-large.png" alt-text="Screenshot of Apple App Store Connect API details.":::
+To add your Apple account to Visual Studio:
 
-    Provided that your account details are valid, your Apple Developer Account will be added to Visual Studio.
+1. In Visual Studio, go to **Tools > Options > Xamarin > Apple Accounts** and click on the **Add** button:
 
-    Note: Only "Admin" keys will be able to register new Bundle IDs and generate new provisioning profiles. "Developer" keys will not be able to perform these tasks.
+    :::image type="content" source="media/apple-account-management/vs/add-account.png" alt-text="Add an Apple Developer Account to Visual Studio.":::
 
-1. Once your account has been added successfully, you'll see your Apple ID and any teams that your Apple ID is part of:
+1. Provide a descriptive **Name** for your API Key and copy the **Issuer ID** and **Key ID** values from Apple's website into the appropriate text boxes. If you haven't already done so, download the **Private Key** from Apple's website to a safe location and then use the **Browse...** button to select the location of the downloaded private key file.
 
-    :::image type="content" source="media/apple-account-management/vs/account.png" alt-text="Apple Developer Account added to Visual Studio.":::
+    :::image type="content" source="media/apple-account-management/vs/api-key-dialog.png" alt-text="Enter your API Key information.":::
```

Comment:
This image is going to need updating to blank out the values of the Issuer ID, Key ID, and Private Key fields. If you don't, the regular SFI scan of content will flag this image for attention due to being "unsafe". It doesn't matter if you're using fake values, it still needs doing. Blurring values out doesn't cut it - AI can often undo the blur. So you literally have to draw a solid colour rectangle over the values.

Btw, I know this is dumb. It's also the world we now live in.

Example of what I mean: 
![](https://github.com/dotnet/docs-maui/blob/main/docs/mac-catalyst/deployment/media/publish-ad-hoc/keychain-access-developer-certificate.png)

I can do this if required.

Reply:
@davidbritch Ah, yea, these are all fake values (I actually used the same fake values that Apple used in their documentation), but I understand that's an issue.

I can replace this image with one that doesn't have any values filled in - I think that would look better than blocking out the values. I'll replace that in just a moment.